app/javascript/packs/application.js

```diff
@@ -3,12 +3,13 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
+import 'bootstrap'
```

Comment:
Use require method to make it consistent.

Reply:
Done